xcube_sh/store.py

```diff
@@ -547,3 +549,85 @@ def _assert_valid_opener_id(cls, opener_id):
                 f' "{SH_DATA_OPENER_ID}",'
                 f' but got "{opener_id}"'
             )
+
+
+class SentinelHubCdseDataStore(SentinelHubDataStore):
+    """
+    Sentinel HUB implementation of the ``xcube.core.store.DataStore``
+    interface.
+    """
+
+    # def __init__(self):
+    #     super().__init__()
+    def __init__(self, **sh_kwargs):
+        super().__init__(SentinelHub(**sh_kwargs))
+
+    ##########################################################################
+    # DataStore impl.
+
+    @classmethod
+    def get_data_store_params_schema(cls) -> JsonObjectSchema:
```

Comment:
Apply DRY-principle: you can avoid code duplication by first getting the base store's schema:

```
    base_schema = super().get_dataset_store_params_schema()
    return JsonObjectSchema(
        properties={
           **base_schema.properties, 
           <special props go here>
        }, 
        required=base_schema.required, 
        additional_properties=base_schema.additional_properties
    )
```